tools/custom-action/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:23.04
+
+ENV LANG=en_US.UTF-8
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt update -y && \
+  apt install -y locales && \
+  locale-gen en_US.UTF-8 && \
+  sysctl -w kernel.unprivileged_userns_clone=1 && \
+  apt install -y \
+    nodejs \
+    npm \
+    git \
+    sudo \
+    libgtk-3-dev \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    python3-full \
+    python3-pip \
+    rsync
+
+COPY entrypoint.sh prepare.sh .env /
+
+RUN \
+  chmod +x /entrypoint.sh /prepare.sh /.env && \
+  export $(cat /.env | xargs) && \
+  python3 -m pip install --break-system-packages \
+    tree_sitter==$PIP_TREE_SITTER_VERSION && \
+  npm install -g @mermaid-js/mermaid-cli@$NPM_MERMAID_CLI_VERSION && \
+  npx --yes -- @mermaid-js/mermaid-cli@$NPM_MERMAID_CLI_VERSION --version && \
```

Comment:
Do we need `npx --yes` here if `npm install` is doing the work?

Reply:
I didn't try without npx --yes. Checking to see if result is everything being cached on execution.

Reply:
That went well, thank you very much!